application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -271,6 +279,22 @@
       }
     }
   }
+
+  $(document).on('click', '#confirmCacheFlushModal .btn-primary', function(event) {
+    event.preventDefault();
+    const flushButton = $('#confirmCacheFlushModal .btn-primary');
+    const target = flushButton.attr('data-action');
+    $.post(target)
+    .done(function() {
+      $(".operation-result").addClass("successmessage");
```

Comment:
The used class is actually `instance-operation-result`
Also, single quotes should be used for simple strings (here and below)